### PR TITLE
Better logging for Graphsync traversal

### PR DESCRIPTION
**Pull request by aarshkshah1992**

### CODE REVIEW

File: responsemanager/runtraversal/runtraversal.go [hunk 1]
```diff
@@ -4,12 +4,14 @@ import (
 	"bytes"
 	"io"
 
+	"github.com/ipfs/go-graphsync/ipldutil"
+	logging "github.com/ipfs/go-log"
```
Comment (dirkmc) [inline responsemanager/runtraversal/runtraversal.go:8]:
```suggestion
	logging "github.com/ipfs/go-log/v2"
```

File: responsemanager/runtraversal/runtraversal.go [hunk 2]
```diff
@@ -39,14 +49,17 @@ func RunTraversal(
 				_, err = io.Copy(blockBuffer, result)
 			}
 			if err != nil {
+				logger.Errorf("failed to write to buffer, link=%s, nBlocksRead=%d, err=%s", lnk, nBlocksRead, err)
 				traverser.Error(err)
 			} else {
 				data = blockBuffer.Bytes()
 				err = traverser.Advance(blockBuffer)
 				if err != nil {
+					logger.Errorf("failed to advcance traversal, link=%s, nBlocksRead=%d, err=%s", lnk, nBlocksRead, err)
```
Comment (dirkmc) [inline responsemanager/runtraversal/runtraversal.go:58]:
```suggestion
					logger.Errorf("failed to advance traversal, link=%s, nBlocksRead=%d, err=%s", lnk, nBlocksRead, err)
```

**Review by dirkmc — COMMENTED** [review]
A couple of nits but otherwise looks good.
Should we also add debug logs for when we're about to load a link so that we can see all the CIDs of successfully loaded links?

**Comment by dirkmc** [comment]
Also it seems like TestNetworkErrors/network_error_block_send is failing - I ran it again and the same test failed.

**Comment by aarshkshah1992** [comment]
@dirkmc That test is a flaky. See the same failure on master

```
aarshshah@Aarshs-MacBook-Pro responsemanager % go test -count=100 -failfast -run TestNetworkErrors
go: downloading github.com/ipld/go-ipld-prime v0.9.1-0.20210324083106-dc342a9917db
go: downloading github.com/ipld/go-codec-dagpb v1.2.0
go: downloading github.com/multiformats/go-multihash v0.0.15
go: downloading github.com/minio/sha256-simd v1.0.0
go: downloading golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83
go: downloading github.com/klauspost/cpuid/v2 v2.0.4
go: downloading golang.org/x/sys v0.0.0-20210309074719-68d13333faf2
--- FAIL: TestNetworkErrors (10.01s)
    --- FAIL: TestNetworkErrors/network_error_block_send (10.00s)
        channelassertions.go:72: 
            	Error Trace:	channelassertions.go:72
            	            				channelassertions.go:48
            	            				responsemanager_test.go:1037
            	            				responsemanager_test.go:629
            	Error:      	Not equal: 
            	            	expected: 0
            	            	actual  : 1
            	Test:       	TestNetworkErrors/network_error_block_send
            	Messages:   	should clear request
FAIL
exit status 1
FAIL	github.com/ipfs/go-graphsync/responsemanager	21.346s
aarshshah@Aarshs-MacBook-Pro responsemanager % git status
On branch master
Your branch is up to date with 'origin/master'.

**Review by dirkmc — APPROVED** [review]

**Pull request merged**